### ISSUE TITLE
Configuration file detection is not evident

### ISSUE THREAD

**Issue by Peter2121** [TigerVNC/tigervnc#956]
When I start vncviewer from command line I can use a x.tigervnc file to specify the connection parameters. This file can be created from vncviewer directly ('Save As' button).
The problem is that if I give just `myfile.tigervnc` as command-line option - it is considered as server name (and failed to resolve).
Digging in the code I found the way to use a configuration file as command-line option - it should be `./myfile.tigervnc` I think it is not evident and can be confusing ;)

**Comment by CendioOssman** [comment]
Yeah, this was an explicit choice to avoid mistakes (and attacks) of someone creating a file matching a hostname.

Ideas on how this could be made more user friendly?

**Comment by Peter2121** [comment]
I would prefer to have a separate option for configuration file, so it is impossible to use it at place of host name, for example
`vncviewer --config=myserver.tigervnc --passwd=myserver.passwd`

**Comment by CendioOssman** [comment]
Our goal was to avoid having to specify extra flags, so I'm afraid that method has already been ruled out.
